### ISSUE TITLE
python3 

### ISSUE THREAD

**Issue by fariza** [underdogio/flask-graylog#1]
I guess you are already aware, but it is not python3 compatible

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/local/lib/python3.4/dist-packages/flask_restful/__init__.py", line 271, in error_router
    return original_handler(e)
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.4/dist-packages/flask/_compat.py", line 32, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1479, in full_dispatch_request
    response = self.process_response(response)
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1691, in process_response
    response = handler(response)
  File "/usr/local/lib/python3.4/dist-packages/flask_graylog.py", line 121, in after_request
    for key, value in request.environ.iteritems()
AttributeError: 'dict' object has no attribute 'iteritems'
```

**Comment by brettlangdon** [comment]
Wasn't 100% aware... but good catch! thanks for sharing.

**Comment by brettlangdon** [comment]
@fariza I just released version `v1.1.0` which uses `.items()` instead of `.iteritems()` so you should be good to go :+1:

Let me know if you have any other issues.

**Comment by fariza** [comment]
it's working perfect, thanks
